src/content/structured/get-started/angular.mdx

```diff
@@ -37,7 +37,7 @@ yarn add @ukic/web-components @ukic/fonts
 
 In the `app.modules.ts` file, import the `CUSTOM_ELEMENTS_SCHEMA`.
 
-Or for standalone components, import the `CUSTOM_ELEMENTS_SCHEMA` into the `.ts` file of any component using ICDS components.
+Or for standalone components, import the `CUSTOM_ELEMENTS_SCHEMA` into the `.ts` file of any component using the components on this Design System.
```

Comment:
Think it should be "from this Design System" instead of "on this Design System" here.